book/src/versions/0.15/2023-04-27.md

```diff
@@ -32,21 +50,25 @@ Impressive!
 
 Avatars, banners, icons, and other assets are received as hashes, which we have
 [`ImageHash`] for as a performance improvement in storage. Instead of storing
-hashes as heap-allocated strings, we store them as 16 bytes on the stack.
-The performance of the deserialization and parsing of hashes is now 38% faster.
+hashes as heap-allocated strings, we store them as 16 bytes on the stack. The
+performance of the deserialization and parsing of hashes is now 38% faster.
```

Comment:
The original was 79 characters long, why is this being  moved?

Reply:
It's simpler to always linewrap at 80 characters than to keep in mind the special case of pre-emptively linewrapping if the next line is <80 characters.

That said, I can revert it. I don't have an opinion on this either way. In fact, I find linewrapping at all somewhat annoying.

Reply:
It doesn't really matter. I don't have any opinions on when to wrap as long as the lines don't exceed a certain width. What I would like to avoid is unrelated diffs in changes, but this is fine if you want to keep it.